docs/geode-native-docs/remote-querying/remote-queries.html.md.erb

```diff
@@ -30,74 +30,105 @@ Querying and indexing operate only on remote cache server contents.
+
+A query execution returns its results as either a `ResultSet` or a `StructSet`. 
 
 Query language features and grammar are described in the <%=vars.product_name%> manual at
-[Querying](geodeman/developing/querying_basics/chapter_overview.html). This section describes areas
-that are unique to the native client.
+[Querying](geodeman/developing/querying_basics/chapter_overview.html).
 
 ### <a id="CreatingIndexes"></a>Creating Indexes
 
-An index can provide significant performance gains for query execution. You create and maintain indexes on the cache server.
-
+Indexes can provide significant performance gains for query execution. You create and maintain indexes on the cache server.
 For detailed information about working with indexes configured on a cache server, 
 see [Working with Indexes](geodeman/developing/query_index/query_index.html) in the server's documentation.
 
 ## <a id="RemoteQueryAPI"></a>Remote Query API
 
-This section gives a general overview of the interfaces and classes that are provided by the Query
-package API, and the shortcut methods provided in the Region interface.
-
-### <a id="CreatingAndManagingQueries"></a>Creating and Managing Queries
-
-You create queries on the cache server by obtaining a `QueryService` method and manage them through
-the resulting `Query` object. The `Region` interface has several shortcut query methods.
+This section gives a general overview of the interfaces and classes that are provided by the query package API.
 
 ### <a id="Query"></a>Query
 
-A `Query` is obtained from a `QueryService` method, which is obtained from the cache. The `Query` interface provides methods for managing the compilation and execution of queries, and for retrieving an existing query string.
+A `Query` is obtained from a `QueryService` method, which is obtained from the cache. The `Query`
+interface provides methods for managing the compilation and execution of queries, and for retrieving
+an existing query string.
 
-You must obtain a `Query` object for each new query.
+You must create a `Query` object for each new query.
 
 ### <a id="ExecutingAQuery"></a>Executing a Query from the Client
 
--  Get a pointer to the `QueryService` method. If you are using the pool API, you should obtain the `QueryService` from the pool.
--  Create a `QueryPtr` to a query (C++) or create a query instance (.NET) that is compatible with the OQL specification.
--  Use the `execute` method for the `Query` interface to submit the query string to the cache server. The server remotely evaluates the query string and returns the results to the client.
--  You can iterate through the returned objects as part of the query process.
+The essential steps to create and execute a query are:
 
-## <a id="AccessingCachedData"></a>Accessing Cached Data
+1.  Create an instance of the `QueryService` class. If you are using the pool API, you should obtain the `QueryService` from the pool.
```

Comment:
there is probably a top-level hint/insistence to use Pools, but might be good to cement this as the recommended approach...